src/xdocs/config_filters.xml

```diff
@@ -999,6 +999,221 @@ public class InputTest {
+          and xpath.
+        </p>
+        <p>
+            Attention: This filter only supports single suppression, and will need
+            multiple instances if users wants to suppress multiple violations.
+        </p>
+      </subsection>
+      <subsection name="Properties" id="SuppressionXpathSingleFilter_Properties">
+        <table>
+            <tr>
+                <th>name</th>
+                <th>description</th>
+                <th>type</th>
+                <th>default value</th>
+                <th>since</th>
+            </tr>
+            <tr>
+                <td>files</td>
+                <td>a Regular Expression matched against the file name associated
+                    with an audit event</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>checks</td>
+                <td>a Regular Expression matched against the name of the check
+                    associated with an audit event.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>message</td>
+                <td>a Regular Expression matched against the message of the check associated
+                    with an audit event.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>id</td>
+                <td>a string matched against the ID of the check associated with an audit
+                    event.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>query</td>
+                <td>a string xpath query.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+        </table>
+      </subsection>
+      <subsection name="Examples" id="SuppressionXpathSingleFilter_Examples">
+        <p>
+          To configure to reject CyclomaticComplexity errors
+          for all methods with name sayHelloWorld
+          inside FileOne and FileTwo files:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="files" value="FileOne.java"/&gt;
+  &lt;property name="checks" value="CyclomaticComplexity"/&gt;
+  &lt;property name="query" value="//METHOD_DEF[@text='sayHelloWorld']"/&gt;
+&lt;/module&gt;
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="files" value="FileTwo.java"/&gt;
```

Comment:
please make `files` as regexp `File(One|Two)\.java` and keep only one instance of module.

Reply:
done.